js/src/forum/components/RankingsPage.js

```diff
@@ -73,10 +73,10 @@ export default class RankingsPage extends Page {
                                         <td>
                                             <div className="PostUser">
                                                 <h3 className="rankings-info">
-                                                    <Link href={app.route.user(user)}>
+                                                    <a href={app.route.user(user)}>
                                                         {i < 4 ? avatar(user, { className: 'info-avatar rankings-' + i + '-avatar' }) : ''}{' '}
                                                         {username(user)}
-                                                    </Link>
+                                                    </a>
```

Comment:
This should be a `Link` component. The `Link` component automatically handles internal/external links for you to ensure that unnecessary page refreshes don't happen.

Reply:
Well, that was exactly what I wanted. I don't know why, but there is an issue when you click on a user. The user page is bugged and can't load. Adding a force refresh temporarily fix the issue (waiting a permanent fix from FoF or Flarum team, since I don't know if this is a flarum or gamification bug

Reply:
@davwheat there is an issue with the Link component (try it on your Flarum)